packages/common/src/components/AppHeader/AppHeaderTabs.tsx

```diff
@@ -29,7 +46,15 @@ const AppHeaderTabs: React.FC<Props> = ({ tabs }) => {
   }
 
   const handleClick = (tab: UserTab) => {
-    history.push(tab.url)
+    if (match.url !== tab.url) {
+      dispatch(resetPatientState())
```

Comment:
Why do we reset patient state between navigations?

Reply:
Because otherwise the searched patient will be saved if the user goes from the create page to a list. This is not how it should work. Also, if the user has searched for a patient and then clicks on the Search/create-tab the search page should be showed, not the patient page.

Reply:
I get that we don't want to show the previously searched patient when user navigates to search page again, but don't we want to keep the patient while navigating between lists an certificates?
Also, I realized we get some weird couplings when we reference webcert specific actions from a component in common, we should invert that coupling.

Reply:
You can try it yourself in the test environments but when I navigate: Search with patient -> List -> Search, the patient is reset for the original application

Reply:
Never mind, I thought we needed the patient info on the list views.
Though I pulled the code and tried it out locally and there's a lot of duplicated api calls. I doesn't look like they bring any new data and could probably be cached.
![image](https://user-images.githubusercontent.com/40682617/169817934-8b93ba20-cff9-4c57-94a0-2a43f0e99aff.png)
Also, when I switch to Signed Certificates tab, I get an error while retrieving the list (probably because I haven't backend) and I end up in an infinite loop of api calls. That shouldn't be able to happen.
![image](https://user-images.githubusercontent.com/40682617/169818659-79249007-6669-48cb-8e49-a7a2fb5443e0.png)